programs/eosio-launcher/main.cpp

```diff
@@ -1639,7 +1641,11 @@ launcher_def::launch (eosd_def &instance, string &gts) {
 
   //Always enable the trace_api_plugin on the bios node
   if (instance.name == "bios") {
-    eosdcmd += "--plugin eosio::trace_api_plugin ";
+    if (old_nodeos) {
+      eosdcmd += "--plugin eosio::history_api_plugin  --filter-on \"*\"";
```

Comment:
Python allows the use of the `'` character for string quoting to avoid having to escape quotes.  If a string contains double quotes, the enclosing quotes can be single quotes and no blackslashes are necessary.

Reply:
Wrong file.